### PR TITLE
Implement transfer fallback enabled flag

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2012

### CODE REVIEW

File: src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx [hunk 1]
```diff
@@ -250,11 +250,13 @@ export const EVMDestinationAdaptorProvider = ({
 
   useEffect(() => {
     if (
+      process.env.REACT_APP_TRANSFER_FALLBACK_ENABLED === "true" &&
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx:253]: Let's move this complex condition to a variable with a good name and use it inside the if

File: src/Contexts/Adaptors/SubstrateDestinationAdaptor.tsx [hunk 2]
```diff
@@ -213,8 +213,14 @@ export const SubstrateDestinationAdaptorProvider = ({
   }, [transactionStatus]);
 
   useEffect(() => {
-    if (transactionStatus === "In Transit" && api && !fallback?.started())
+    if (
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/SubstrateDestinationAdaptor.tsx]: Let's move this complex condition to a variable with a good name and use it inside the if

**Review by AndreiNavoichyk — APPROVED** [review]

File: src/Contexts/Adaptors/SubstrateDestinationAdaptor.tsx (hunk 2 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/SubstrateDestinationAdaptor.tsx]: Fixed

File: src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx (hunk 1 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx:253]: Fixed

**Pull request merged**